material/problems/Kinematics/SwingingShip/SwingingShip.tex

```diff
@@ -24,48 +24,110 @@
   Hence, the angular motion of the tire is used to swing the ship back and
   forth.
   %
-  \\[0.5pc]
+  \\[0.45pc]
   During a certain interval, the \textbf{counter-clockwise} rotation rate of
-  the tire is described by $\omega_A = 10*\sin(t)$.
-  The radius of the tire is $R_A = 0.3$ m.
-  %
-  \\[0.5pc]
+  the tire is described by $\omega_A = 10\ast\sin(t)$.
   Let the \textbf{counter-clockwise} rotation rate of the ship be denoted by
-  $\omega_B$. The distance from $B_o$ to the bottom surface of the ship is
-  $R_B = 6$ m.
+  $\omega_B$.
+  \\[0.45pc]
+  The radius of the tire is $R_A = 0.3$ m, and the distance from $B_o$ to the
+  bottom surface of the ship is $R_B = 6$ m.
   %
-  \\[0.5pc]
-  Right-handed orthogonal unit vectors \uvecxyz{n} are fixed in \basis{N}
-  as shown.
+  %\\[0.45pc]
+  %Right-handed orthogonal unit vectors \uvecxyz{n} are fixed in \basis{N}
+  %as shown.
   %
-%  \begin{center}
-%    \includegraphics[height=6cm]{mayflower.jpg}
-%    \hspace{1cm}
-%    \includegraphics[height=6cm]{boat.png}
-%  \end{center}
+  \begin{center}
+    \includegraphics[height=6cm]{mayflower.jpg}
+    \hspace{1cm}
+    \includegraphicsAB[height=6cm]{boat-solution.png}{boat.png}
+  \end{center}
   %
-  %\\[0.5pc]
-  \begin{itemize}
+  \begin{enumerate}
     \item \textbf{Calculate} the velocity of the point of the tire in contact
-      with the ship, $\vel{A_B}{N}$.
-      %
-    \item \textbf{Calculate} the velocity of the point of the ship in contact
-      with the tire, $\vel{B_A}{N}$.
+      with the ship, $\vel{A_B}{N}$, and calculate the velocity of the point of
```

Comment:
Everywhere that there is A_B and B_A please replace with `\contactPoint{A}{B}` and `\contactPoint{B}{A}`. This macro didn't exist before but I've started using it recently.